orion/controllers/programs/programsController.go

```diff
@@ -81,6 +84,29 @@ func DeleteProgram(c *gin.Context) {
   return
 }
 
-func CheckValidProgram() bool {
+func CheckValidProgram(program Program) bool {
```

Comment:
So at the end of our session, Tony and I talked about how instead of returning a `bool`, `CheckValidProgram(...)` will now return an `error` type. Booleans funnel results into two values, but it would be good to know why a program is not valid. Take a look at his change to see how to use `error`.